backend/src/main/java/ubb/courses/backend/services/security/SecurityService.java

```diff
@@ -39,4 +39,12 @@ public boolean isTeacher() {
                 authority -> authority.getAuthority().equals(AuthorityType.TEACHER.name())
         ).count() == 1;
     }
+
+    @Override
+    public boolean isStudent() {
+        val authorities = this.getAuthorities();
+        return authorities.stream().filter(
+                authority -> authority.getAuthority().equals(AuthorityType.STUDENT.name())
+        ).count() == 1;
```

Comment:
I think .anyMatch would be a better approach than filtering and counting. Could you please change this everywhere ?

Reply:
Done ^^